lib/openxml/elements/run_properties.rb

```diff
@@ -0,0 +1,47 @@
+module OpenXml
+  module Elements
+    class RunProperties < OpenXml::Element
+      namespace :a
+      tag :rPr
+
+      attribute :italic, displays_as: :i, expects: :boolean
+      attribute :bold, displays_as: :b, expects: :boolean
+      attribute :size, displays_as: :sz, one_of: (100..400_000).to_a
+      attribute :strike, one_of: %w[sngStrike dblStrike noStrike]
+      attribute :underline, displays_as: :u, one_of: %w[
+        dash
+        dashHeavy
+        dashLong
+        dashLongHeavy
+        dbl
+        dotDash
+        dotDashHeavy
+        dotDotDash
+        dotDotDashHeavy
+        dotted
+        dottedHeavy
+        heavy
+        none
+        sng
+        wavy
+        wavyDbl
+        wavyHeavy
+        words
+      ]
+
+      def initialize(*boolean_properties, **value_properties)
```

Comment:
You were showing me these yesterday, I like the `*boolean_properties` and `**value_properties` 👍